RELEASE/scripts/CONSUME.ash

```diff
@@ -662,23 +693,31 @@ void handle_special_items(Diet d, OrganSpace space, OrganSpace max)
 
 	if (d.within_limit($skill[Sweat Out Some Booze]))
 	{
-		Consumable sweat;
-		sweat.sk = $skill[Sweat Out Some Booze];
-		sweat.organ = ORGAN_NONE;
-		sweat.cleanings[0] = new OrganCleaning(ORGAN_LIVER, daily_limit($skill[Sweat Out Some Booze]));
-		d.handle_organ_cleanings(sweat, space, max);
-		while(d.within_limit($skill[Sweat Out Some Booze]))
-			d.add_action(sweat.to_action(d));
+		float sweatOutSomeBoozeValue = cleaned_liver_value(daily_limit($skill[Sweat Out Some Booze]), d.nightcap);
+		if(sweatOutSomeBoozeValue > 0)
+		{
+			Consumable sweat;
+			sweat.sk = $skill[Sweat Out Some Booze];
+			sweat.organ = ORGAN_NONE;
+			sweat.cleanings[0] = new OrganCleaning(ORGAN_LIVER, daily_limit($skill[Sweat Out Some Booze]));
+			d.handle_organ_cleanings(sweat, space, max);
+			while(d.within_limit($skill[Sweat Out Some Booze]))
+				d.add_action(sweat.to_action(d));
```

Comment:
Hmm. I know you're just indenting this here, but note to self: Make sure this isn't a bug. It kinda looks like it'd think it's clearing 3 liver 3 times instead of 1 liver 3 times when you have 3 uses left...

Reply:
I'll test that tomorrow. I wrote that code originally on PR #28

Reply:
Looks fine to me. I added some debugging print statements. The while loop adds 3 casts of Sweat out some booze to the diet which get consolidated into a `cast 3 Sweat Out Some Booze` by the diet optimisation.
Mojo filters work similarly as they're the only other organ cleaning that usable more than once per day.